Sources/Datadog/CrashReporting/CrashContext/CrashContext.swift

```diff
@@ -150,47 +150,35 @@ private struct CodableUserInfo: Codable {
 
 private struct CodableRUMViewEvent: Codable {
     private let model: RUMViewEvent
-    private let attributes: [String: Encodable]
-    private let userInfoAttributes: [String: Encodable]
+    private let errorAttributes: [AttributeKey: AttributeValue]?
```

Comment:
I guess it's only for sanity now, as this will be removed in `RUMM-1463` as we're saying in this new comment:
```swift
// RUMEvent.swift

/// Error attributes. Only set when `DM == RUMErrorEvent` and error describes a crash.
/// Can be entirely removed when RUMM-1463 is resolved and error values are part of the `RUMErrorEvent`.
let errorAttributes: [String: Encodable]?
```
Right? Meaning, `RUMM-1463` will make this crash-context-coding thing even cleaner ✨.

Reply:
Ah yes, I forgot about that one 👍

Reply:
So, I cannot remove `CodableRUMViewEvent` as I originally thought because `RUMViewEvent` is not Decodable. And if  we make it Decodable, it means that we need to encode the error attributes...

Reply:
Right 👍. We will do it in `RUMM-1584` (_Remove `RUMEvent` container._) 👌.